src/components/CreatePacking.tsx

```diff
@@ -0,0 +1,251 @@
+import * as React from "react";
+import * as actions from "../actions"
+import { connect } from "react-redux";
+import { ErrorMessage, StoreState } from "src/types";
+import { Dispatch } from "redux";
+import strings from "src/localization/strings";
+import { Grid, Form, Button, Select, DropdownItemProps, DropdownProps, Input, InputOnChangeData, Loader } from "semantic-ui-react";
+import { DateInput } from 'semantic-ui-calendar-react';
+import { FormContainer } from "./FormContainer";
+import { Packing, Product, PackageSize, PackingState } from "famifarm-typescript-models";
+import LocalizedUtils from "src/localization/localizedutils";
+import * as moment from "moment";
+import Api from "../api";
+import { Redirect } from "react-router";
+
+export interface Props {
+    keycloak?: Keycloak.KeycloakInstance;
+    onPackingCreated?: (packing: Packing) => void,
+    onError: (error: ErrorMessage) => void
+}
+
+export interface State {
+    productId?: string,
+    packingStatus?: PackingState,
+    packageSizeId?: string,
+    packedCount: number,
+    products: Product[],
+    loading: boolean,
+    packageSizes: PackageSize[],
+    date: string,
+    redirect: boolean,
+    packingId?: string
+}
+
+class CreatePacking extends React.Component<Props, State> {
```

Comment:
Fix indentation on this file. It is indentet with 4 spaces and the rest of the project with 2 spaces

Reply:
resolved